tests/models/pvt/test_modeling_pvt.py

```diff
@@ -317,14 +317,13 @@ def test_inference_fp16(self):
         r"""
         A small test to make sure that inference work in half precision without any problem.
         """
-        model = PvtForImageClassification.from_pretrained(
-            "Zetatech/pvt-tiny-224", torch_dtype=torch.float16, device_map="auto"
-        )
+        model = PvtForImageClassification.from_pretrained("Zetatech/pvt-tiny-224", torch_dtype=torch.float16)
```

Comment:
just removing `device_map="auto"` and not to work on `_no_split_modules`. Hope this is acceptable.

Reply:
If it fits, that works for me!